spec/lib/packet-spec.js

```diff
@@ -24,12 +24,13 @@ describe("Packet", function() {
         magic: 1669485411,
         options:
         {
-            dhcpMessageType: { value: 3, name: 'DHCPREQUEST' },
+            dhcpMessageType: 0x2,
```

Comment:
Shouldn't we use the enum value here (https://github.com/RackHD/on-dhcp-proxy/blob/master/lib/dhcp-protocol.js#L100), rather than raw?